app/src/main/java/fulguris/settings/preferences/DomainPreferences.kt

```diff
@@ -161,7 +161,7 @@ class DomainPreferences constructor(
      * - Ask: Ask the user whether or not to launch the app
      */
     var launchAppOverride by preferences.booleanPreference(R.string.pref_key_launch_app_override, false)
-    var launchAppLocal by preferences.enumPreference(R.string.pref_key_launch_app, NoYesAsk.NO)
+    var launchAppLocal by preferences.enumPreference(R.string.pref_key_launch_app, NoYesAsk.ASK)
```

Comment:
Default should be set to Yes, see: https://github.com/Slion/Fulguris/issues/598#issuecomment-1985950169

Reply:
Okay